cernopendata/modules/fixtures/data/docs/cod-welcome/cod-welcome.json

```diff
@@ -11,7 +11,7 @@
             {"primary": "News" }
         ],
         "author": "Open Data Portal team",
-        "created": "November 15th 2017",
+        "created": "2017-11-17",
```

Comment:
Guessing this was changed intentionally from 15 November to 17 November, correct?

Reply:
yup :) closer to the release

Reply:
... that would have been `2017-12-17` 😄

Reply:
omg yes 😂